lib/bootstrap_form/inputs/base.rb

```diff
@@ -31,9 +31,16 @@ def bootstrap_select_group(field_name)
           bootstrap_alias field_name
         end
 
+        # Creates the methods *_without_bootstrap and *_with_bootstrap.
+        #
+        # If your application did not include the rails gem for one of the dsl
+        # methods, then a name error is raised and suppressed. This can happen
+        # if your application does not include the actiontext dependency due to
+        # `rich_text_area` not being defined.
         def bootstrap_alias(field_name)
           alias_method "#{field_name}_without_bootstrap".to_sym, field_name
           alias_method field_name, "#{field_name}_with_bootstrap".to_sym
+        rescue NameError
```

Comment:
I'd prefer to do the RuboCop exception specifically on the line.
```suggestion
        rescue NameError # rubocop:disable Lint/SupressedException
```